package.json

```diff
@@ -120,7 +120,6 @@
     "jwt-decode": "^4.0.0",
     "morgan": "^1.10.0",
     "nocache": "^4.0.0",
-    "npm": "^10.4.0",
```

Comment:
don't know why this was ever included (can't see it was ever in the template), but brought in security vulns so removed